styles.css

```diff
@@ -522,6 +516,121 @@ footer ul {
   align-self: flex-end;
 }
 
+#overlay {
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  background-color: black;
+  opacity: 0.8;
+  backdrop-filter: blur(5px);
+  pointer-events: all;
+  display: none;
+}
+
+.modal {
+  border: 1px solid black;
+  z-index: 1000;
```

Comment:
- [ ] On the mobile version I would also suggest that you try to make your design as close to the Figma design as much as possible. 
- Add a class to the `div` container containing the `p tag` and the buttons. that would make it easy for you to align the container with the image.
- Make sure the button text is aligned with the button icon. Include icons for both buttons

> Your Design
![mobilfig](https://user-images.githubusercontent.com/71140133/232128173-f22d9769-d986-4aa7-acee-e3ebb3067174.PNG)

> Figma Design
![mobilReal](https://user-images.githubusercontent.com/71140133/232128242-7c3192da-a417-47db-8b8e-744034b73831.PNG)